### PR TITLE
sap_hana_install: Use polling for hdblcm

### PR DESCRIPTION
**Pull request by berndfinger**
Solves issue #804.

@marcelmamula - Thanks for reporting, proposing a possible solution, and initial testing!

### CODE REVIEW

**Review by marcelmamula — APPROVED** [review]
LGTM, let me know once it is merged.

**Review by sean-freeman — APPROVED** [review]
lgtm 👍

**Pull request merged**
